symengine/cwrapper.h

```diff
@@ -201,6 +208,9 @@ CWRAPPER_OUTPUT_TYPE complex_imaginary_part(basic s, const basic com);
 CWRAPPER_OUTPUT_TYPE complex_double_real_part(basic s, const basic com);
 //! Assign to s, an imaginary double where com is a complex double
 CWRAPPER_OUTPUT_TYPE complex_double_imaginary_part(basic s, const basic com);
+//! Extract the real and imaginary doubles from the std::complex<double> stored
+//! in basic
+CWRAPPER_OUTPUT_TYPE complex_double_get(dcomplex *d, basic s);
```

Comment:
Instead of using pointers, you can return the value. 

    dcomplex complex_double_get(const basic s);